src/framework/components/render/component.js

```diff
@@ -198,6 +199,11 @@ class RenderComponent extends Component {
         var app = this.system.app;
         var scene = app.scene;
 
+        if (this._initialRootBone) {
+            this.rootBone = this._initialRootBone;
+            this._initialRootBone = null;
+        }
```

Comment:
This feels odd. Why the need for initial root bone? Anim component does not have it. It does not feel like a proper fix to me.

Reply:
Updated based on our discussion. The rootBone property is now set along with other initial component properties. However the cloning of skin instances which used to occur on the setting of the root bone has now been deferred to when the component is enabled.